### PR TITLE
:up: `@parcel/watcher@2.5.0`

### PR DESCRIPTION
**Pull request by bpasero**

### CODE REVIEW

File: build/.moduleignore [hunk 1]
```diff
@@ -110,6 +110,9 @@ node-pty/third_party/**
 @parcel/watcher/src/**
 !@parcel/watcher/build/Release/*.node
 
+# exclude parcel watcher prebuilds
+@parcel/watcher-**
```
Comment (deepak1556) [inline build/.moduleignore:114]: given we remove them at install stage, is this needed ?

File: extensions/package.json [hunk 2]
```diff
@@ -10,13 +10,13 @@
...
   "devDependencies": {
-    "@parcel/watcher": "2.1.0",
+    "@parcel/watcher": "2.5.0",
     "esbuild": "0.23.0",
     "vscode-grammar-updater": "^1.1.0"
   },
   "overrides": {
     "node-gyp-build": "4.8.1",
-    "@parcel/watcher@2.1.0": {
+    "@parcel/watcher@2.5.0": {
```
Comment (deepak1556) [inline extensions/package.json:19]: This override can be removed, 2.5.0 watcher version comes with the required node-addon-api.

File: package.json [hunk 3]
```diff
@@ -70,7 +70,7 @@
   "dependencies": {
     "@microsoft/1ds-core-js": "^3.2.13",
     "@microsoft/1ds-post-js": "^3.2.13",
-    "@parcel/watcher": "2.1.0",
+    "@parcel/watcher": "2.5.0",
```
Comment (deepak1556) [inline package.json:73]: Commenting at the wrong line due to github UI,

But there is a similar override in this file which can be removed.

File: remote/package.json [hunk 4]
```diff
@@ -44,7 +44,7 @@
     "kerberos@2.1.1": {
       "node-addon-api": "7.1.0"
     },
-    "@parcel/watcher@2.1.0": {
+    "@parcel/watcher@2.5.0": {
       "node-addon-api": "7.1.0"
     }
```
Comment (deepak1556) [inline remote/package.json:49]: Override can be removed

File: build/.moduleignore (hunk 1 above)
Comment (bpasero) [reply build/.moduleignore:114]: Ok lets remove it.

**Review by deepak1556 — APPROVED** [review]
Great, thanks! one less override 🚀

**Pull request merged**